client/containers/Pub/PubMeta/CitationsModal.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
This is mostly copied from V5's `PubOptionsCite` component, with the removal of version-related code and a server call.